packages/devtools_app/lib/src/screens/memory/panes/allocation_profile/allocation_profile_table_view.dart

```diff
@@ -338,7 +339,13 @@ class _AllocationProfileTableControls extends StatelessWidget {
         ),
         const SizedBox(width: denseSpacing),
         RefreshButton.icon(
-          onPressed: allocationProfileController.refresh,
+          onPressed: () async {
+            ga.select(
+              analytics_constants.memory,
+              analytics_constants.MemoryEvent.gc,
```

Comment:
We should not be re-using the same event twice. This event is already used for the GC button.